tests/trace/test_scored_calls_query.py

```diff
@@ -0,0 +1,225 @@
+from __future__ import annotations
+
+import pytest
+
+import weave
+from tests.trace.util import client_is_sqlite
+from weave.scorers import Scorer
+from weave.trace.op import Op
+from weave.trace.weave_client import WeaveClient
+from weave.trace_server.trace_server_interface import CallsQueryReq, CallsQueryStatsReq
+
+
+def assert_number_of_calls(client: WeaveClient, count: int):
+    stats = client.server.calls_query_stats(
+        CallsQueryStatsReq(
+            project_id=client._project_id(),
+        )
+    )
+    assert stats.count == count
+
+
+def output_field_for_name(name: str) -> str:
```

Comment:
These seem like essentially the same methods you added in `feedback_types.py`?

Reply:
Yes, but i intentionally did not not want rely on that implementation since it is part of what I was testing